### ISSUE TITLE
Adopt Icon to FF theme

### ISSUE THREAD

**Issue by 8sd** [RESTEDClient/RESTED#130]
I'm using the default Firefox dark theme.
Unfortunately the Icon and the background have nearly the same color. So I cannot see the icon.

The color of the ff dark theme icons is: #b2b2b2

**Comment by esphen** [comment]
Yes, I agree.

This would require creating a new gray icon using gimp using the file in the [img directory](https://github.com/RESTEDClient/RESTED/tree/master/dist/img)

A new manifest key is also required. See: https://developer.mozilla.org/en-US/Add-ons/WebExtensions/manifest.json/browser_action

**Comment by hdwilber** [comment]
I would like to fix this one, is still open? @esphen 
Is possible to enable the Assign option for issues? So that we won't step on ourselves.
Thanks

**Comment by esphen** [comment]
@hdwilber I made you a collaborator, that should give you privilige to assign issues as you see fit. Feel free to work on this if you like!
